docker_utils/rtenvsub.sh

```diff
@@ -398,7 +399,9 @@ function mirror_envsubst_paths {
             color_echo magenta "Destination directory does not contain any files, no pipes created for ${full_path}!"
         else
             for file in "${files[@]:-}"; do
-                add_on_sig "rm -f ${destination}${file#${full_path}}"
+                if [ "${DEV_MODE}" = true ] ; then
```

Comment:
You don't need to use comparison [] to evaluate true/false since these are commands, you can just do:

if {binary_variable} ; then